tests/app/main/helpers/test_suppliers.py

```diff
@@ -75,6 +75,25 @@ def test_returns_expected_value_for_input(self, g12_recovery_supplier_ids, expec
             assert is_g12_recovery_supplier('123456') is expected_result
 
 
+class TestG12RecoveryDrafts(BaseApplicationTest):
+    @pytest.mark.parametrize(
+        'draft_ids_config, expected_result',
+        [
+            (None, set()),
+            ('', set()),
+            (42, set()),
+            ('12:32', set()),
+            ([123456, 789012], set()),
+            ('123456', {123456}),
+            ('123456,789012', {123456, 789012}),
```

Comment:
Is it worth testing what happens if a stray space or newline gets in there?

Reply:
Good idea - added.